### PR TITLE
feat(core): have separate buttons for expanding and collapsing section breaks

### PR DESCRIPTION
**Pull request by DeeMysterio**

### CODE REVIEW

File: frappe/public/js/frappe/form/toolbar.js [hunk 1]
```diff
@@ -181,31 +181,27 @@ frappe.ui.form.Toolbar = Class.extend({
...
+		let section_fields = me.frm.meta.fields.filter((field) => field.fieldtype == "Section Break");
+		// Expand all sections
+		if(section_fields.length != 0) {
+			this.page.add_menu_item(__("Expand Sections"), function () {
+				for (let section of section_fields) {
+					me.frm.get_field(section.fieldname).collapse(false);
+				}
+			}, true);
+			this.page.add_menu_item(__("Collapse Sections"), function () {
+				let section_fields = me.frm.meta.fields.filter((field) => field.fieldtype == "Section Break");
```
Comment (Alchez) [inline frappe/public/js/frappe/form/toolbar.js]: Remove this line. You've already declared it on line 184.

File: frappe/public/js/frappe/form/toolbar.js [hunk 2]
```diff
@@ -181,31 +181,27 @@ frappe.ui.form.Toolbar = Class.extend({
 				}, true);
 			}
 		}
-
-		// Request feedback
-		if(!this.frm.doc.__unsaved) {
-			if(is_submittable && docstatus == 1) {
-				this.page.add_menu_item(__("Request Feedback"), function() {
```
Comment (Alchez) [inline frappe/public/js/frappe/form/toolbar.js]: I think you accidentally deleted this button.

**Review by Alchez — CHANGES_REQUESTED** [review]

**Review by Alchez — APPROVED** [review]

**Comment by Alchez** [comment]
@DeeMysterio, can you make this upstream?

**Comment by DeeMysterio** [comment]
> @DeeMysterio, can you make this upstream?

Did this [here](https://github.com/frappe/frappe/pull/9025)

**Pull request merged**